TruePath/AbsolutePath.cs

```diff
@@ -54,6 +54,16 @@ private AbsolutePath(string value, bool checkAbsoluteness)
     /// <inheritdoc cref="IPath.FileName"/>
     public string FileName => Underlying.FileName;
 
+    /// <inheritdoc cref="IPath.StartsWith"/>
+    public bool StartsWith(AbsolutePath other) => Value.StartsWith(other.Value);
+
+    /// <inheritdoc cref="IPath.IsPrefixOf"/>
+    public bool IsPrefixOf(AbsolutePath other)
+    {
+        if (!(Value.Length <= other.Value.Length && other.Value.StartsWith(Value))) return false;
+        return other.Value.Length == Value.Length || other.Value[Value.Length] == Path.DirectorySeparatorChar;
```

Comment:
I don't think `other.Value[Value.Length] == Path.DirectorySeparatorChar` is ever true for normalized paths (except for the bare root `/`), is it? So maybe let's drop that check?

Reply:
solved in [63a457e](https://github.com/ForNeVeR/TruePath/pull/69/commits/63a457eeec87e47de9b5fd8d119b2ec6e1990212)